scavenger-api/src/main/kotlin/com/navercorp/scavenger/dto/ExportSnapshotMethodDto.kt

```diff
@@ -0,0 +1,33 @@
+package com.navercorp.scavenger.dto
+
+import com.navercorp.scavenger.entity.ExportSnapshotMethodEntity
+
+data class ExportSnapshotMethodDto(
+    val filterInvokedAtMillis: String,
+    val packages: String,
+    val status: String,
+    val excludeAbstract: String,
+    val parent: String,
+    val signature: String,
+    val type: String,
+    val usedCount: String,
+    val unusedCount: String,
+    val lastInvokedAtMillis: String
+) {
+    companion object {
+        fun from(entity: ExportSnapshotMethodEntity): ExportSnapshotMethodDto {
+            return ExportSnapshotMethodDto(
+                filterInvokedAtMillis = entity.filterInvokedAtMillis.toString(),
+                packages = entity.packages ?: "",
```

Comment:
```suggestion
                packages = entity.packages.orEmpty(),
```
The other fields below are also available.

Reply:
I fixed this review on commit 7d70bf1.